src/components/InputComponent/InputComponent.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { InputComponent } from './InputComponent';
+import { Meta, Story } from '@storybook/react';
+
+export default {
+  title: 'Input',
+  component: InputComponent,
+  argTypes: {
+    border: {
+      description: 'Add rounded border or remove it',
+    },
+    dark: {
+      description: 'Set dark theme',
+    },
+  },
+} as Meta;
+
+const Template: Story = (args: any) => <InputComponent {...args} />;
+export const Default = Template.bind({});
+Default.args = {
+  border: 'normal',
+};
+
+export const Dark = Template.bind({});
+Dark.args = {
+  border: 'normal',
+  dark: true,
+};
```

Comment:
Please add stories for error & disabled states as per the [requirement](https://github.com/igorizviekov/mt-booking_app/issues/18)

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/d1ed5fdefd0af9a627399bf0ab1541da866ae781